.github/workflows/Build.yml

```diff
@@ -55,21 +55,24 @@ jobs:
       id: docker-image-cache
       with:
         path: kphp-build-env-${{matrix.os}}.tar
-        key: docker-image-cache-${{matrix.os}}-${{ hashFiles('.github/workflows/Dockerfile.*', 'tests/python/requirements.txt') }}
+        key: kphp-build-env-${{matrix.os}}-${{ hashFiles('.github/workflows/Dockerfile.*', 'tests/python/requirements.txt') }}.tar
 
     - name: Build and save docker image
       if: steps.docker-image-cache.outputs.cache-hit != 'true'
       run: |
-        docker build -f $GITHUB_WORKSPACE/.github/workflows/Dockerfile.${{matrix.os}} $GITHUB_WORKSPACE -t kphp-build-img-${{matrix.os}}
-        docker save --output kphp-build-env-${{matrix.os}}.tar kphp-build-img-${{matrix.os}}
+        docker build -f $GITHUB_WORKSPACE/.github/workflows/Dockerfile.${{matrix.os}} $GITHUB_WORKSPACE \
+              -t kphp-build-img-${{matrix.os}} \
+              --cache-from=type=local,src=kphp-build-img-${{matrix.os}}-cache \
+        docker tag kphp-build-img-${{matrix.os}} kphp-build-img-${{matrix.os}}-cache \
```

Comment:
The `\` is only needed to move **ONE** command across multiple lines.

Reply:
ok, remove